.overcommit.yml

```diff
@@ -41,6 +41,7 @@ PreCommit:
     description: "Checking Rails best practices"
     exclude:
       - "**/__snapshots__/*"
+      - "**/**_controller.rb"
```

Comment:
question: why remove controllers from rails best practices?

Reply:
Because all of our controllers get falsely flagged as having "unused" functions

Reply:
I wasn't able to replicate the false flags, I'm on `rails_best_practices v1.23.2`

Reply:
same - it only processes edited files. Did you try editing the file and then running?

Reply:
> Did you try editing the file and then running?

Yeah, I reset the commits in this branch locally, then staged them again, and was able to commit without getting warnings from overcommit.